xtask/src/tidy.rs

```diff
@@ -392,7 +392,7 @@ impl TidyDocs {
             )
         }
 
-        let poorly_documented = ["hir_expand", "mbe", "parser", "hir_ty"];
+        let poorly_documented = ["hir_ty"];
```

Comment:
@flodiebold would you like to document the `hir_ty` yourself ^^ ?

Reply:
I guess I should 😬